jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -28,6 +28,18 @@
       <g-file-import id="file-uploader"></g-file-import>
     </plugin-file-import-select>
 
+    <v-row v-if="catalog_selected === 'Gaia'">
+      <v-text-field
+        v-model.number="max_rows"
+        type="number"
+        step="10"
+        :rules="[() => max_rows!=='' || 'This field is required']"
+        label="Max rows"
+        hint="Maximum amount of rows."
+        persistent-hint
+      ></v-text-field>
+    </v-row>
+
```

Comment:
Describing the number of sources as "rows" makes sense when you're thinking about the catalog tables, but from the UI it's a bit disconnected from the data structure. Let's use "sources" instead?

```suggestion
    <v-row v-if="catalog_selected === 'Gaia'">
      <v-text-field
        v-model.number="max_sources"
        type="number"
        step="10"
        :rules="[() => max_sources!=='' || 'This field is required']"
        label="Max sources"
        hint="Maximum number of sources."
        persistent-hint
      ></v-text-field>
    </v-row>

```